src/components/Modals/Nfts/NftModal.tsx

```diff
@@ -63,12 +64,17 @@ const NftTab: React.FC<TabProps> = props => {
 }
 
 export type NftModalProps = {
-  nftItem: NftItem
+  nftAssetId: AssetId
 }
 
-export const NftModal: React.FC<NftModalProps> = ({ nftItem }) => {
+export const NftModal: React.FC<NftModalProps> = ({ nftAssetId }) => {
```

Comment:
`<NftModal />` is "rendered" calling `nft.open()` so props never change - this would never re-render on NFT update